ansible/configs/ibm-rhoic/software.yml

```diff
@@ -18,7 +18,34 @@
   tags:
     - actual_sausage_making
   tasks:
-    ## Authenticated alredy from infrastructure creation
+    ## Authenticating
+    - name: Authorize to get bearer access and refresh tokens
+      uri:
+        url: "{{ ibm_cloud_api_iam_url }}"
+        headers:
+          Authorization: Basic Yng6Yng=
+        method: POST
+        body_format: form-urlencoded
+        body:
+          grant_type: "urn:ibm:params:oauth:grant-type:apikey"
+          response_type: "cloud_iam"
+          apikey: "{{ sandbox_master_api_key }}"
```

Comment:
We need to transition this to use the SID API key that is created vs the master key. The master key should really only be used to configure access to the account and for administrators to get in if necessary.

Reply:
Commit a change right now to fix which key is used.